README.md

```diff
@@ -22,78 +23,86 @@ Here's a recipe template.
 | Estimated IBU | 30.05 |
 | Estimated Color | ![#000000](https://placehold.it/15/000000/000000?text=+) `#000000` |
 
-### Yeast: Safale S-04
-
-### Fermentables
+### Yeast/Bacteria
+| Type | Name |
+|---|---|
+| Yeast | Safale S-04 |
+| Bacteria | OYL-605 |
 
+### Mash Bill
 | Amount | Fermentable | L° | Bill % |
 |---|---|---|---|
-
-<!-- | 8 lb | Maris Otter | 2.3 | 69.6% |
+<!--
+| 8 lb | Maris Otter | 2.3 | 69.6% |
 | 2 lb | Light Munich | 8 | 17.4% |
 | 12 oz | Caramel / Crystal 80L | 80 | 6.5% |
-| 12 oz | Special Roast | 50 | 6.5% | -->
-
-### Adjuncts
-
-| Amount | Ingredient | Bill % |
-|---|---|---|
+| 12 oz | Special Roast | 50 | 6.5% |
+-->
 
-<!-- OR -->
-<!-- No non-fermentable adjuncts added to this recipe. -->
-
-### Hop Schedule
-
-| Amount | Variety | Type | AA | Use | Time | IBU | Bill % |
-|---|---|---|---|---|---|---|---|
-
-<!-- | 1 oz | Northern Brewer | Pellet | 9.3 | Boil | 30 min | 24.02 | 33.3% |
-| 1 oz | Styrian Goldings | Pellet | 5.5 | Boil | 10 min | 6.7 | 33.3% |
-| 1 oz | Fuggles | Pellet | 4.5 | Boil | 5 min | 3.01 | 33.3% | -->
-
-### Water Profile
+### Hops
+| Amount | Variety | AA | Bill % |
+|---|---|---|---|
+<!-- 
+| 1 oz | Northern Brewer | 9.3 | 33.3% |
+| 1 oz | Styrian Goldings | 5.5 | 33.3% |
+| 1 oz | Fuggles | 4.5 | 33.3% |
+-->
+
+### Water Chemistry
+| Ion | ppm |
+|---|---|
+| Ca+2 | 75 |
+| Mg+2 | 5 |
+| SO4-2 | 150 |
+| Na+ | 10 |
+| Cl- | 50 |
+| HCO3- | 0 |
 
-#### Requirements (1.XX qt/lb)
+## Steps (5 gal)
 
+### Salts
+| Name | Amount |
+|---|---|
+<!--
+| Gypsum | 4.5g |
+| CaCl2 | 1.3g |
+| Epsom | 0.9g |
+| NaCl | 0.5g |
+-->
+
+### Mash
 | Amount | Description | Temp | Time |
 |---|---|---|---|
-
-<!-- | 7.4 gal | Total | N/A | N/A |
+<!--
+| 7.4 gal | Total | N/A | N/A |
 | 3.8 gal | Strike | 152 °F | 60 min |
-| 3.6 gal | Fly Sparge | 170 °F | 30 min | -->
-
-#### Salts
-
-| Amount | Name |
-|---|---|
+| 3.6 gal | Fly Sparge | 170 °F | 30 min |
+| 1.5 gal | Rinse | Room |
+-->
 
-<!-- | 3.8 g | Calcium Chloride (anhydrous) |
-| 1.5 g | Epsom Salt |
-| 3.8 g | Gypsum |
-| 3 g | Baking Soda | -->
+1. Do a thing
+1. Do a second thing
+1. Final thing
 
-### Steps
-
-#### Mash
-
-- [ ] Do a thing
-- [ ] Do a second thing
-- [ ] Final thing
-
-#### Pre-Boil
```

Comment:
There is some stuff that may be documented before a boil, like gravity/pH measurements and kettle volume.

Reply:
That should be in the brew day template and not the recipe.